src/haz3lweb/UpdateAction.re

```diff
@@ -10,6 +10,7 @@ type settings_action =
   | Dynamics
   | Benchmark
   | ContextInspector
+  | Breadcrumb_bar(int, bool)
```

Comment:
It's best to stick to the case convention for constructors of the other files and call it `BreadcrumbBar` instead of Breadcrumb_bar

Reply:
Thanks! I didn't notice that